skycoin/skywire
# Update UI dependencies

Did you run `make format && make check`?
No code was changed

 Changes:	
- Angular was updated from v12.2.5 to v12.2.13. Several nested dependencies were also updated.

How to test this PR:
Run `make build-ui` or at least `make install-deps-ui` to make sure the dependencies are updated. After recompiling the UI, it should continue working well.
